src/server/gc_worker/compaction_filter.rs

```diff
@@ -287,14 +287,12 @@ struct WriteCompactionFilter {
     gc_scheduler: Scheduler<GcTask<RocksEngine>>,
     // A key batch which is going to be sent to the GC worker.
     mvcc_deletions: Vec<Key>,
-    // The count of records covered the current mvcc-deletion mark. The mvcc-deletion
-    // mark will be sent to the GC worker only if `mvcc_deletion_overlaps` is 0. It's
-    // a little optimization to reduce modifications on write CF.
-    mvcc_deletion_overlaps: Option<usize>,
+    // The mvcc-deletion mark will be sent to the GC worker only if `mvcc_del_gc_needed` is
+    // not none. It's a little optimization to reduce modifications on write CF.
+    mvcc_del_gc_needed: Option<bool>,
```

Comment:
It's impossible to be `Some(false)` now? Then, is it necessary to make it an `Option`?